tools/SetupFlow/DevHome.SetupFlow/Views/LoadingView.xaml

```diff
@@ -32,6 +32,23 @@
                             </Setter.Value>
                         </Setter>
                     </Style>
+                    <!-- Because a data template is an encapsulation layer, dynamic style lookups won't look past the data template.
+                         This means that using a converter to change foreground to a ThemeResource on a textblock does not work.
+                         The work around is to define the styles in a resource and use the keys in the data template.-->
+                    <Style x:Key="PrimaryTextStyle" TargetType="TextBlock">
```

Comment:
Can we instead place the converter inside the `DataTemplate`? Maybe inside the `DataTemplate.StackPanel.Resources`?

Reply:
Done.